src/app/Application/ApplicationAssets.tsx

```diff
@@ -187,7 +193,7 @@ If network is the problem, you can use the command line tool to accomplish the t
                     />
                 </FormGroup>
                 <Button variant="secondary" type="submit">
-                    Upload Asset
+                    Upload {isCustomization ? 'Customization' : 'Asset'}
```

Comment:
Do we want to call it "customizations", since that is how we are referring everywhere else.

Reply:
We are using both